src/pages/History.tsx

```diff
@@ -0,0 +1,29 @@
+import { withAction } from "../store";
+import { ActionState } from "../types";
+
+export const Histroy = withAction(({ action }: { action: ActionState[] }) => {
```

Comment:
Explicitly declare the params (use interface) being accepted here and in the withAction function.
`const History: React.FC<HistoryActionProps> = ({ action }) => {}`

Tip: Separate declaration of React Function Component and its usage in different lines for code clarity
`export const History = withAction(HistoryInner)`